src/sentry/static/sentry/app/views/projectChooser.jsx

```diff
@@ -21,6 +21,13 @@ class ProjectChooser extends React.Component {
   };
 
   componentWillMount() {
+    const {organization} = this.props;
+
+    const tasks = getOnboardingTasks(organization);
+    this.task = tasks.find(
```

Comment:
Is it possible this component re-renders because the `task` URL param changes? If that's the case then `this.task` becomes stale. I think it's safer to refactor this into a method and just have it recalculate each time. If you wanted to further optimize it, we could memoize the method so that it caches the result based on the input (but that's not necessary here).